standards/core/mix.md

```diff
@@ -0,0 +1,128 @@
+The pool size of mix nodes SHOULD be large enough for the mixing to be effective.
+We RECOMMEND a pool size of at least 100 mix nodes for the mixing to be effective.
+
+The serialized [Waku Message](https://rfc.vac.dev/waku/standards/core/14/message) SHOULD be the payload in the [sphinx packet](https://rfc.vac.dev/vac/raw/mix#4-sphinx-packet-format).
+
+In order to provide anonymity of the sender node from the `service-node`,  `Single Use reply blocks` or `anonymous replies` as specified in the original [sphinx](#references) paper SHALL be used.
+
+A node that sends messages using `mix` MAY use two redundant (paths)[https://rfc.vac.dev/vac/raw/mix#24-node-discovery] to have better reliability of the message being delivered.
+It is up to the higher-layer mixed protocol to deduplicate redundant messages received in this way.
+
+## Node Roles
+
+Mix protocol defines 3 roles for the nodes in the mix network - `sender`, `exit`, `intermediary`.
+
+- An `sender` node is the originator node of a message, i.e a node that wishes to publish/query messages to/from the waku network.
+- An `exit` node is responsible for delivering messages to destination peer in the network.
+- An `intermediary` node is responsible for forwarding a mix packet to the next mix node in the path.
+
+A Waku relay node SHOULD by default have mix `intermediary` and `exit` node roles in the network.
+The implementation MAY provide a configuration to disable a node from acting as an `intermediary\exit` node.
+
+Any waku node that wishes to publish/query messages from the waku network MUST act as a `sender` node.
+
+Resource-restricted/Edge nodes with short connection windows SHOULD always be acting as `sender` nodes.
+
+## ENR updates
+
+Each waku node that supports `mix` SHOULD indicate the same in its discoverable [ENR](https://github.com/waku-org/specs/blob/master/standards/core/enr.md).
+The following fields MUST be set as part of the discoverable ENR of a mix waku node:
+- The `bit 5` in the [waku2 ENR key](https://github.com/waku-org/specs/blob/master/standards/core/enr.md#waku2-enr-key) is reserved to indicate `mix`support. This bit MUST be set to true to indicate `mix` support.
+- A new field `mix-key` SHOULD be set to the `ed25519 public key` which is used for sphinx encryption.
+
+### Discovery
+
+Mix protocol provides better anonymity when an originator node has a sufficiently large pool of mix nodes to do path selection.
+This moves the problem into discovery domain and requires the following from discovery mechanisms:
+1. It is important for nodes to be able to discover as many nodes as possible quickly. This becomes especially important for edge nodes that come online just to publish/query messages for a short period of time.
+2. It is important to have the most recent online status of the nodes so that mix paths that are selected are not broken which lead to reliability issues.
```

Comment:
Would that be the reason we add a TTL or timestamp to ENRs? :thinking:

Reply:
Would we want to use rendezvous in edge nodes? cc @SionoiS

Reply:
> Would we want to use rendezvous in edge nodes? cc @SionoiS

Yes it make sense since it's very lively. It takes care of the discovery problem it's just not very decentralized.